### DISCUSSION TITLE
Девайс-кроссинг: надо или нет?

### DISCUSSION THREAD

**Discussion by NetBUG** [b4ck5p4c3/0x08.in#368] [Ideas]
Судя по раз в день-два всплывающим постам "Отдам ХХХ в сильные и заботливые руки", спейс притягивает людей, которым есть, что предложить другим материальное. Более того, иногда а) возникают запросы на что-то, б) в Спейсе появляются материальные активы, которые не привязаны к грядущим задачам, занимают место и могут быть переданы тем, кто может их быстро использовать и повысить общественное благосостояние.

Одним из участников была создана беседа "Бекспейс-барахолка", которая, однако, была осуждена как снижающая уровень восприятия Спейса.
В связи с этим есть ряд вопросов:
1) Надо ли это вообще?
2) Если да, аффилировать ли напрямую со спейсом или пусть живёт своей жизнью?
3) Как назвать?
4) Где и как хостить? На каждый девайс issue на гитхаб не заведёшь

**Comment by ahsima1** [comment]
Та беседа была создана без какого либо обсуждения и НЕ участником, поэтому к ней и возникли вопросы.

**Comment by heeeyflo** [comment]
>  1. Надо ли это вообще?

Мне кажется, это дичайше круто — reuse/upcycling, все дела, но только если оно происходит именно в формате свопа, максимально безответственного свопа, где самая высокая цена — двухлитровка томатного сока или плитка Риттер-спорта.

Подмешивание товаро-денежных отношений может скверно обернуться для нас (заплатил Х фантиков за старую PS3, а она оказалась нерабочей, Бэкспейс херня, раз не чекают/модерируют барахолку), да и просто непонятно чем спейсу может быть полезна такая площадка.

>     2. Если да, аффилировать ли напрямую со спейсом или пусть живёт своей жизнью?

Почему бы и нет. Соощество вокруг спейса испытывает потребность в такой площадке, а социальный капитал бренда поможет площадке быстрее набрать критическую массу людей. 

Такой хакерно-хардверно-диайвайно-ориентированный своп, набрав критическую массу, может стать популярным и привлечь новых людей в сообщество или увеличить энгейджмент, история вроде:

 — «блин, клевая штука»
 — «ага, урвал на бэкспейсном свопе за бутылку сидра»
 — «крутяк, слышал что-то о них, ребята сидят в подвале и канифоль плавят 24/7»
 — «да, классные челы, бывал у них разок на ХХХ, хочешь ссылку кину?»
 — «давай»

>     3. Как назвать?

Нейминг — это боль.

>     4. Где и как хостить? На каждый девайс issue на гитхаб не заведёшь

Кажется, что формат канала с комментариями впишется просто идеально, но так как выдавать доступ на публикацию всем подряд будет сложно, предлагаю сделать так:

1. Публичный канал с комментариями для объявлений (посты) и обсуждений (комментарии)
2. Простенький бот для управления объявлением (в MVP: добавить объявление, пометить объявление как закрытое).

**Reply by GRbit** [reply]
На мой взгляд история про "Бэкспейс херня" встретиться в 100% случаев и если этого бояться, то можно и вообще ничего не сделать.

**Comment by GRbit** [comment]
Поддерживаю идею, поделился своими мыслями в https://github.com/b4ck5p4c3/0x08.in/discussions/370 но слишком поздно)

Делаю копипейст.

Большая часть из нас любит и уважает всякие хорошо сделанные технические вещи. Уверен у каждого со временем накапливается куча вещей которые с одной стороны "хорошая же штука, ценная", с другой стороны никакого применения в быту не находят годами. Либо это какая-то модифицированная вещь, которая для обывателя "ой фу, вы разбирали, вдруг она сломается теперь?", а для своих это может быть реально нужный важный мод. Напимер встроить в какой-нибудь прибор питаемый от 9v порт type c для подключения к павербанку.

Мне было бы приятно расставаться с такими вещами зная что они окажутся в руках достойного человека. Надеюсь в этом плане надйутся единомышленники, которые как и я готовы расстаться с вещью подешевле, рассказать про какие-то доработки/допилы, нюансы изготовления и другие проблемы, при условии что вещь пойдёт к человеку с руками от плечей. Да и просто приятно помочь брату технократу с приобретением нужной ему вундервафли)

Основные поинты барахолки:

1.    Администрация ответственности ни за что не несёт.
 2.   Подразумевается, что участник предлагающий что-то сообществу делает это на эксклюзивных условиях. Например на авито у него объявление о продаже старого ноута висит за 20к, а кому-то из сообщества готов уступить за 10к.
3.    Можно и нужно предлагать модифицированные вещи со своими костылями дополняющими/исправляющими оригинальную конструкцию.
4.    Предприятие позиционируется как некомерческое, скорее отбить расходы. Типа купил поломанную штуку, починил, продаю за то за что купил + запчасти.

Следить за этим на мой взгляд бесполезно, всегда найдётся участник который в первую очередь будет желать заработка, так что вопросы саморегулирования предлагаю отдать на откуп сообществу.

Ещё мысль: некоторые лоты по желанию участников можно продавать в формате аукциона. Владелец говорит "хочу с этого 1к рублей". Хакспейс промоутит аукцион, а участники в течении скажем недели могут повышать ставки. Всё что выше 1к идёт на аренду хакспейса)

**Comment by hitriy** [comment]
Пришел в сообщество именно с целью отдать несколько плат интересующимся. Думаю барахолка очень нужна, чтобы поменяться интересным барахлом, а то и купить/продать что-нибудь. Чата в телеге будет достаточно. Будет барахолка — отдам собранный, но не прошитый HydraBus=)
